### ISSUE TITLE
Provide error messages returned from the App Center CLI in the Cake build log

### ISSUE THREAD

**Issue by steverichey** [cake-contrib/Cake.AppCenter#22]
## Expected Behavior
This is a recommended improvement. When building via Cake, we often receive the error "AppCenter: Process returned an error (exit code 3)." The App Center CLI provides more detailed error information, but this information is not visible in the Cake output log, confounding troubleshooting efforts.

## Current Behavior
Instead of the vague error message, read the message from standard error or standard output and display.

## Possible Solution
I'm not sure how the parser that generates this repository works, but I'm guessing there would be a change there.

## Steps to Reproduce (for bugs)
The exit code 3 error happens intermittently when uploading to App Center. You can also see this message when attempting to call `AppCenterAppsShowWithResult` with an invalid app name.

## Context
This is causing CI builds to fail.

## Your Environment
* Module version used: Cake 0.38.5, Cake.AppCenter 1.3.0, App Center CLI 2.7.3
* Operating System and PowerShell version: macOS 10.15.7, noted in both bash and zsh

**Comment by MihaMarkic** [comment]
Yep, I need to figure out how to display errors. Right now they are not displayed, because an error is thrown before.
